### PR TITLE
[FlashScript] version 2.1.5

### PR DESCRIPTION
**Pull request by andyblac**
only show Sys Temp if key exists
allow more time for device to flash and reboot, some stock device's can be slow to perform a flash.
add device count for flash failed devices, only show flashed / failed when in flashing mode.
fix varSeq in version parse.
fix forced version flashing for stock firmware.

### CODE REVIEW

File: tools/flash-shelly.py [hunk 1]
```diff
@@ -473,7 +478,7 @@ def parse_info(device_info, action, dry_run, quiet_run, silent_run, mode, exclud
     latest_fw_label = f"{RED}No {device_info.variant} available{NC}"
     flash_fw_version = '0.0.0'
     dlurl = None
-  elif not dlurl or durl_request.status_code != 200:
+  elif not dlurl or durl_request.status_code != 200 or ('Content-Type' in durl_request.headers and durl_request.headers['Content-Type'] != 'application/zip'):
```
Comment (rojer) [inline tools/flash-shelly.py:481]: durl_request.headers.get('Content-Type', '') != 'application/zip'

File: tools/flash-shelly.py [hunk 2]
```diff
@@ -443,7 +448,7 @@ def parse_info(device_info, action, dry_run, quiet_run, silent_run, mode, exclud
   colour_mode = device_info.colour_mode
   dlurl = device_info.dlurl
   flash_label = device_info.flash_label
-  sys_temp = device_info.info['sys_temp'] if 'sys_temp' in device_info.info else '0'
+  sys_temp = device_info.info['sys_temp'] if 'sys_temp' in device_info.info else None
```
Comment (rojer) [inline tools/flash-shelly.py:451]: sys_temp = device_info.info.get('sys_temp', None)

File: tools/flash-shelly.py [hunk 3]
```diff
@@ -487,7 +492,8 @@ def parse_info(device_info, action, dry_run, quiet_run, silent_run, mode, exclud
     logger.info(f"{WHITE}Device ID: {NC}{device_id}")
     logger.info(f"{WHITE}IP: {NC}{wifi_ip}")
     logger.info(f"{WHITE}Model: {NC}{model}")
-    logger.info(f"{WHITE}Sys Temp: {NC}{sys_temp}˚c{NC}")
+    if sys_temp:
```
Comment (rojer) [inline tools/flash-shelly.py:495]: if sys_temp is not None

it's unlikely , but we could have system temperature of 0 degrees

Comment (andyblac) [reply tools/flash-shelly.py]: done

File: tools/flash-shelly.py (hunk 2 above)
Comment (andyblac) [reply tools/flash-shelly.py]: done

File: tools/flash-shelly.py (hunk 1 above)
Comment (andyblac) [reply tools/flash-shelly.py]: done

**Pull request merged**